server/views/pages/pop/index.njk

```diff
@@ -30,33 +30,83 @@
 }) }}
 {% endif %}
 
+{% if scenario == "pending" %}
+{{ mojAlert({
+  variant: "warning",
+  title: "Account pending verififcation",
+  showTitleAsHeading: true,
+  dismissible: false,
+  html: 'Your account has been created and is will be verified at your next face to face appointment with your probation practitioner.
+  <br /> Once your account is activated you will be able to acccess the links below.'
+}) }}
+{% endif %}
+
+{% if session.popVerificationStatus == "pending" %}
```

Comment:
This has been handled as a scenario, not using session in the staff side of this work. This cannot be session as the staff user and the pop user will be different and may be researched as different sessions.

Reply:
Workaround added